components/forms/WeightForm.js

```diff
@@ -0,0 +1,60 @@
+import styled from "styled-components";
+
+export default function WeightForm({ onAddWeight }) {
+  function sendForm(event) {
+    event.preventDefault();
+    const formData = new FormData(event.target);
+    const { weight, date } = Object.fromEntries(formData);
+
+    onAddWeight(weight, date);
+  }
+  return (
+    <WeightFormElement aria-label="Add weight and date" onSubmit={sendForm}>
+      <input type="number" name="weight" min="0" step="0.001" />
```

Comment:
You do not use `label` for your `input` elements. For a better accessibility you should add label elements. [See MDN.](https://developer.mozilla.org/en-US/docs/Learn/Forms/How_to_structure_a_web_form#the_label_element) As far as I know the `aria-label` for the form element is not enough to have an accessible form. 🤔

Reply:
good point, changed that